opm/material/densead/Evaluation.hpp

```diff
@@ -255,9 +268,9 @@ class Evaluation
     Evaluation operator-() const
     {
         Evaluation result;
-        result.value_ = -this->value_;
-        for (unsigned varIdx = 0; varIdx < size; ++varIdx)
-            result.derivatives_[varIdx] = - this->derivatives_[varIdx];
+        // set value and derivatives to negative
```

Comment:
> I guess it is to avoid exposing temporary-creating operations, if I remember correctly?

yeah, that's true for `operator-()`, but not for `operator==()`. anyway, I don't think this is important enough to invest (the probably significant) effort to get it into Dune. In particular since we need to work around this issue anyway for several years to come.